.github/workflows/reusable_terraform_plan_apply.yml

```diff
@@ -148,4 +165,50 @@ jobs:
           payload: |
             {"blocks":[{"type": "section","text": {"type": "mrkdwn","text": ":no_entry: Failed GitHub Action:"}},{"type": "section","fields":[{"type": "mrkdwn","text": "*Workflow:*\n<${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}|${{ github.workflow }}>"},{"type": "mrkdwn","text": "*Job:*\n${{ github.job }}"},{"type": "mrkdwn","text": "*Repo:*\n${{ github.repository }}"}]}]}
         env:
-          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+          SLACK_WEBHOOK_URL: ${{ env.SLACK_WEBHOOK_URL }}
+  member-account-ram-association:
```

Comment:
Minor comment about jobs in the reusable workflow. If you extract common code from a number of workflow (in this case member-account-ram-association), but not all workflows, maybe it is OK for that code to be a separate reusable workflow, leaving the original reusable workflow only for the common to all. I know it is possible to feed inputs and turning it on and off that way, but the reusable workflow becomes not common anymore and it may be more difficult to read it, when it becomes a one monolithic reusable workflow.